src/fat32.c

```diff
@@ -28,14 +28,14 @@ long mega65_fat32_create_contiguous_file(char* name, long size,
     unsigned int fat_offset = 0;
     int j;
 
-    clusters = size / 4096;
+    clusters = (unsigned short)(size / 4096);
```

Comment:
why not uint16_t?

Reply:
Because `clusters` is still `unsigned short`. I agree that stdint types would be better, but I'd prefer to refactor this in a separate PR, and preferable when a test is in place. Here the idea is to convert implicit casts to explicit ones.